apk-generator/v2/app/templates/index.html

```diff
@@ -68,6 +68,16 @@
                 <span id="status-message" data-original="Building your customized android app"></span>
             </div>
 
+	    <div class="col-md-offset-3 col-xs-offset-1 col-xs-10 col-md-6" style="margin-top: -10px;">
+   	        <center>
+                    <div id="status" style="margin-bottom:10px;"></div>
+                    <a id="log-message" style="cursor:pointer;"> -Logs- </a>
+		    <pre id = "build-log" style = "display:none;">
+		    </pre>
```

Comment:
just saw that i missed space in this part. will send the correct version once the entore pr is over.